apps/src/code-studio/components/ModelManagerDialog.jsx

```diff
@@ -81,7 +81,7 @@ export default class ModelManagerDialog extends React.Component {
           handleClose={this.closeModelManager}
           useUpdatedStyles
         >
-          <h2>A.I Trained Models</h2>
+          <h2>A.I. Trained Models</h2>
```

Comment:
Any chance we can add this string to i18n too?

Reply:
Thank you for looking at this! I added it to i18n :)